### PR TITLE
Append dependencies licenses

### PR DESCRIPTION
**Pull request by radarhere**
Suggestion for https://github.com/python-pillow/Pillow/issues/4907

Adds a DEPENDENCIES_LICENSE file, that is appended onto Pillow's LICENSE file.

### CODE REVIEW

**Comment by hugovk** [comment]
Thanks for the PR!

I suggest putting the licences as separate files with .txt extensions, in some subdirectory, to make them easier to inspect/update individually in the repo. 

A command can still cat them all together at the end of the Pillow licence.

File: dependency_licenses/FREETYPE2.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+The  FreeType 2  font  engine is  copyrighted  work and  cannot be  used
+legally  without a  software license.   In  order to  make this  project
+usable  to a vast  majority of  developers, we  distribute it  under two
+mutually exclusive open-source licenses.
+
+This means  that *you* must choose  *one* of the  two licenses described
+below, then obey  all its terms and conditions when  using FreeType 2 in
+any of your projects or products.
+
+  - The FreeType License, found in  the file `FTL.TXT', which is similar
```
Comment (nulano) [inline dependency_licenses/FREETYPE2.txt:10]: Should the files `FTL.TXT` and `GPLv2.TXT` referenced by FreeType be included as well?

Comment (radarhere) [reply dependency_licenses/FREETYPE2.txt:10]: I don't know - the reader is instructed to pick one of them, so if we include both, then we're including a license that doesn't necessarily apply... which I think could be confusing.

**Comment by hugovk** [comment]
Thanks!

File: dependency_licenses/FREETYPE2.txt (hunk 1 above)
Comment (hugovk) [reply dependency_licenses/FREETYPE2.txt:10]: It should be easy enough for people to find those files if they need them, and let's keep things simpler. We can add them if necessary later.

**Pull request merged**